day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,66 @@
+#Ten little monkeys
```

Comment:
Good job on this monkey challenge Tyler! Can you think of a way to DRY this up? That stands for `Don't repeat yourself`, and it is a common theme in programming. Any time you're repeating something, there is probably a way to do more by writing less. Maybe you could use a loop that executes 10 times? I like what you did with the array and using the index. See if you can refactor this to execute within some kind of loop, so that you have less lines of code.